### DISCUSSION TITLE
External plugins disabled?

### DISCUSSION THREAD

**Discussion by jklemm235** [jgraph/drawio#3715] [Q&A]
I used an external plugin (https://github.com/tobiashochguertel/c4-draw.io.git) to create a diagram, now there doesnt seem to be support on apps.diagram.net anymore for external diagrams and some parts are now showing anymore, anybody got an idea how I can still see my diagram in a nice way?

**Comment by davidjgraph** [comment]
We removed external plugin support a long time ago on app.diagrams.net. That specific plugin breaks the data model we use, using it is not good.

You can enable plugins in the desktop tool.

**Reply by jklemm235** [reply]
Nevermind seems to be a different problem sorry, seems to be that the C4 Relationship arrows are buggy in my diagram
sorry

**Comment by ghost** [comment]
For the Windows desktop version, external plug-in extensions have been disabled. How can I enable them again?
